addons/trenches/functions/fnc_placeConfirm.sqf

```diff
@@ -29,46 +29,36 @@ GVAR(digPFH) = -1;
 call EFUNC(interaction,hideMouseHint);
 
 [_unit, "DefaultAction", _unit getVariable [QGVAR(Dig), -1]] call EFUNC(common,removeActionEventHandler);
-
 _unit setVariable [QGVAR(isPlacing), false, true];
 
 // Delete placement dummy and create real trench
 params ["_unit"];
 if (isNull GVAR(trench)) exitWith {};
 
+private _trenchClass = typeOf GVAR(trench);
+private _vecDirAndUp = [vectorDir GVAR(trench), vectorUp GVAR(trench)];
+private _pos = getPosWorld GVAR(trench);
 deleteVehicle GVAR(trench);
-private _trench = createVehicle [GVAR(trenchClass), [0, 0, 0], [], 0, "NONE"];
 
-GVAR(trenchPlacementData) params ["_dx", "_dy", "_offset"];
-private _basePos = GVAR(trenchPos);
-private _angle = (GVAR(digDirection) + getDir _unit);
+private _trench = createVehicle [_trenchClass, _pos, [], 0, "CAN_COLLIDE"];
```

Comment:
Bit weird to spawn the object at a different z, but the same x,y. I fear this will look strange in MP, especially because this is a static object with slow position updates. Why not spawn at [0,0,0]?